recipe/run_test.bat

```diff
@@ -1,8 +1,8 @@
-cd /D "%SRC_DIR%"
+cd /D nb_conda_kernels
 
 conda clean --lock
 conda install -y --quiet -n root -c conda-forge notebook
-conda install -y --quiet -n _test -c r r-irkernel
+conda install -y --quiet -p %CONDA_PREFIX% -c r r-irkernel
```

Comment:
@bollwyvl this test is broken with latest `conda`/`conda-build`. My attempts to fix that did not work. Do you know what must be done to fix it? Or should we just remove that and trust the upstream CI? Everything do seem to be BTW, I tested this locally and it worked.